esx/PXELinux.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: PXELinux
+name: ESXi PXELinux
+oses:
+- ESXi 5
+- ESXi 5.5
+- ESXi 6
+%>
+
+DEFAULT linux
+LABEL linux
+KERNEL undionly.kpxe
+APPEND dhcp && chain <%= foreman_url('iPXE') %>
```

Comment:
FWIW, I use the following snippet to go straight from PXE to kickstart as I am not running iPXE

```
DEFAULT install
NOHALT 1
LABEL install
 KERNEL boot/esxi60u2/mboot.c32
 APPEND -c boot/esxi60u2/boot.cfg ks=<%= foreman_url("provision")%>
 IPAPPEND 2
```
At least for ESXI 6.0